### PR TITLE
Reconfigure settings based on updates to react reanimated

### PR DESCRIPTION
**Pull request by lohannon**

### CODE REVIEW

**Comment by lohannon** [comment]
https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/installation/#android

**Review by townofdon — APPROVED** [review]
Thanks for this addition! A couple of code cleanup suggestions but this looks like it matches the recommended code changes for `react-native-reanimated`.

File: templates/blimp/android/app/src/main/java/com/blimp/MainApplication.java [hunk 1]
```diff
@@ -33,6 +35,11 @@ public boolean getUseDeveloperSupport() {
         protected String getJSMainModuleName() {
           return "index";
         }
+
+        @Override
+        protected JSIModulePackage getJSIModulePackage() {
+            return new ReanimatedJSIModulePackage(); // <- add
```
Comment (townofdon) [inline templates/blimp/android/app/src/main/java/com/blimp/MainApplication.java]: Small nit: recommend removing the code comment `// <- add`

File: templates/jet/android/app/src/main/java/com/helloworld/MainApplication.java [hunk 2]
```diff
@@ -33,6 +35,11 @@ public boolean getUseDeveloperSupport() {
         protected String getJSMainModuleName() {
           return "index";
         }
+
+        @Override
+        protected JSIModulePackage getJSIModulePackage() {
+            return new ReanimatedJSIModulePackage(); // <- add
```
Comment (townofdon) [inline templates/jet/android/app/src/main/java/com/helloworld/MainApplication.java]: Small nit: recommend removing the code comment `// <- add`

**Pull request merged**